app/views/layouts/_content.html.haml

```diff
@@ -63,6 +63,9 @@
             = render :partial => "layouts/listnav"
           = yield :left
         #custom_left_cell
+- elsif @angular_app_name
+  .container-fluid{:style => "overflow: hidden; height: 100%;", 'data-ng-app' => "#{@angular_app_name}"}
+    %div{'ui-view' => ''}
 - else
```

Comment:
The form itself is the right side too, so output-wise there is no difference between the ```_content``` approach or the ```form``` approach. We just have to pick one. I personally prefer the form itself because it just looks cleaner.

```ui-router``` , as one can imagine, cannot route to the non-angular areas of the app. So when transitioning from an angular to a non-angular screen, we'll just have to use techniques like ```$window.location.href = /controller/explorer```